apps/fluent-tester/src/TestComponents/Test.tsx

```diff
@@ -58,6 +63,18 @@ const styles = StyleSheet.create({
 const isMobile = Platform.OS == 'android' || (Platform.OS == 'ios' && !Platform.isPad);
 
 export const Test = (props: TestProps): React.ReactElement<Record<string, never>> => {
+  const [showStatus, setShowStatus] = React.useState(false);
+  const theme = useTheme();
+  const fontFamily = theme.typography.families.primary;
+
+  const plusCodepoint = 0x2795;
```

Comment:
can we use prettier fluent icons directly?

Reply:
W can't use SVGs because they won't work properly on windows.. so maybe if pngs are available?

Reply:
Pngs aren't available but pdfs are so I will try that.

Reply:
Yeah pngs generally aren't great since they don't scale with DPI, so to handle that gracefully we would need to add assets for each DPI level.

It /is/ a test app so probably not as important though.

Personally I think this is fine since it adds a test case for font icons

Reply:
Offline Saad had mentioned font icons for windows and the fluent svg icon for other platforms. The latest commit added that so any opinions?